index.html

```diff
@@ -0,0 +1,114 @@
+                <p><u>Isaac Jung</u> is based in Seoul, Republic of Korea.
+                    A flexible creator skilled in conceptualizing visual systems, analyzing product UIUX,
+                    sound editing, and site specific sonic installations He is also a traveler,
+                    an avid barista, a part-time composer, a devoted table tennis player,
+                    and an eager learner in all the arts.</p>
+            </div>
+            <div class="content02">
+                <h2>Experience</h2>
+                <h3>K: The Portfolio Lab­­—Rebranding Freelance</h3>
+                <p class="dates">SEP, 2018–Oct, 2018</p>
+                <p>Rebranded the studio’s logotype, business cards, welcoming brochures, website, and letterhead.
+                </p>
+                <h3>THE VIGIL—Electronic Music Performer</h3>
+                <p class="dates">APR, 2017 & APR, 2018</p>
+                <p>Performed at the all night music festival that MICA hosts annually
+                </p>
+                <h3>ALEX Berlin—Guest Performer</h3>
+                <p class="dates">MAR, 2018</p>
+                <p>As a member of MICA's SoundArt residency at Maze Kreuz Berg, performed on a live broadcast of
+                    electronic
+                    music performance on Germany’s FM Radio and online.
+                </p>
+                <h3>K: The Portfolio Lab­­—Instructor for Ideation</h3>
+                <p class="dates">JUN, 2017–AUG, 2017</p>
+                <p>Created, structured, and taught: 8 week curriculum for expansive ideation and concept development for
+                    rising freshmen at various art institutes in the US.
+                </p>
+            </div>
+            <div class="content03">
+                <h2>Skills</h2>
+                <h3>Graphic Design</h3>
+                <ul class="u-list-reset">
+                    <li>Concept Ideation</li>
+                    <li>Typeface Design</li>
+                    <li>Social Media Branding</li>
+                    <li>Editorial & Publication</li>
+                    <li>Digital Photography</li>
+                    <li>Narrartive Wireframing</li>
+                </ul>
+                <h3>Sound Design</h3>
+                <ul class="u-list-reset">
+                    <li>Studio Recording</li>
+                    <li>Mixing & Mastering</li>
+                    <li>Stage Setting</li>
+                    <li>Live Performance</li>
+                    <li>Sound Installation</li>
+                    <li>Foley & Sound Effects</li>
+                </ul>
+            </div>
+            <div class="content04">
+                <h2>Language</h2>
+                <h3>Bilingual: Enlgish/Korean</h3>
+                <h2>Education</h2>
+                <h3>Maryland Institute College of Art(MICA)</h3>
+                <p class="dates">AUG, 2016–Current</p>
+                <p>BFA candidate in Graphic Design<br>Expected May 2022
```

Comment:
The `br` element generally should be reserved for meaningful line breaks, like in an address or poem; see [MDN's docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br) for more info.

Instead of relying on `br` here (or line 74), I'd instead consider either multiple `p` elements or using a list.